test/search.test.js

```diff
@@ -182,3 +182,45 @@ test('finds 👨‍👩‍👦', (t) => {
   const found = search('family')
   t.equal(found.items.filter(i => i.title === 'family man, woman, boy').length, 1)
 })
+
+test('does not throw when emojiDetails is undefined', (t) => {
+  t.plan(1)
+  t.equal(alfredItem(undefined), undefined)
+})
+
+test('alfredItems() does not throw when operationType is omitted', (t) => {
+  t.plan(0)
+  alfredItems(['😀'])
```

Comment:
We should use actual assertions instead of relying on exceptions to bubble up. This is more explicit and would provide better diagnostic information if an unexpected exception does ever occur.

```suggestion
  t.plan(1)
  t.doesNotThrow(() => alfredItems(['😀']))
```